deploy/examples/common.yaml

```diff
@@ -67,29 +79,14 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshots/status"]
     verbs: ["update"]
-  - apiGroups: ["storage.k8s.io"]
-    resources: ["volumeattachments"]
-    verbs: ["get", "list", "watch", "update", "patch"]
-  - apiGroups: ["storage.k8s.io"]
-    resources: ["volumeattachments/status"]
-    verbs: ["patch"]
-  - apiGroups: [""]
-    resources: ["nodes"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: [""]
-    resources: ["persistentvolumeclaims/status"]
-    verbs: ["update", "patch"]
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: "psp:rook"
+  name: 'psp:rook'
   labels:
     operator: rook
     storage-backend: ceph
-    app.kubernetes.io/name: rook-ceph
-    app.kubernetes.io/instance: rook-ceph
-    app.kubernetes.io/component: ceph-csi
```

Comment:
There are several instances of these labels being removed, and they all seem to be on non-CSI resources, so this seems correct to me.